UITests/DomainAutocompleteTests.swift

```diff
@@ -21,6 +21,14 @@ class DomainAutocompleteTests: KIFTestCase {
         ensureAutocompletionResult(textField, prefix: "www.yahoo.com/", completion: "")
         tester().clearTextFromFirstResponder()
 
+        tester().enterTextIntoCurrentFirstResponder("www.yah")
+        ensureAutocompletionResult(textField, prefix: "www.yah", completion: "oo.com/")
+        tester().enterTextIntoCurrentFirstResponder("\u{0008}")
```

Comment:
Cool, glad you could find a way to enter that pesky delete character :)

Since I would have no idea what `\u{0008}` means without studying this code, could you extract this into a `deleteCharacterFromFirstResponder` (or something like that) method? You could even put this in `Global.swift` if you want, since we might want to use this in other places.